man/waybar-hyprland-workspaces.5.scd

```diff
@@ -48,7 +50,8 @@ Additional to workspace name matching, the following *format-icons* can be set.
 		"active": "",
 		"default": ""
 	},
-	"sort-by-number": true
+	"all-outputs": false,
+        "show-special": false
```

Comment:
Can you please explain in doc what it is?

Reply:
done